galsim/base.py

```diff
@@ -1887,30 +1911,56 @@ class Kolmogorov(GSObject):
+    other GSObject dimensions.  Assuming that the other sky coordinates you are using are all in
+    arcsec (e.g. the pixel scale when you draw the image, the size of the galaxy, etc.), then you
+    should convert this to arcsec as well:
 
         >>> lam = 700  # nm
         >>> r0 = 0.15 * (lam/500)**1.2  # meters
         >>> lam_over_r0 = (lam * 1.e-9) / r0  # radians
         >>> lam_over_r0 *= 206265  # Convert to arcsec
+        >>> psf = galsim.Kolmogorov(lam_over_r0)
+
+    To make this process a bit simpler, we recommend instead providing the wavelength and Fried
+    parameter separately using the parameters `lam` (in nm) and `r0` (in m).  GalSim will then
+    convert this to any of the normal kinds of angular units using the `scale_unit` parameter:
+
+        >>> psf = galsim.Kolmogorov(lam=lam, r0=r0, scale_unit=galsim.arcsec)
 
-    The FWHM of the Kolmogorov PSF is ~0.976 lambda/r0 (e.g., Racine 1996, PASP 699, 108).
+    When drawing images, the scale_unit should match the unit used for the pixel scale or the WCS.
+    e.g. in this case, a pixel scale of 0.2 arcsec/pixel would be specified as `pixel_scale=0.2`.
 
-    A Kolmogorov can be initialized using one (and only one) of three possible size parameters:
-    `lam_over_r0`, `fwhm`, or `half_light_radius`.  Exactly one of these three is required.
+    A Kolmogorov object may also be initialized using `fwhm` or `half_light_radius`.  Exactly one
+    of these four ways to define the size is required.
+
+    The FWHM of the Kolmogorov PSF is ~0.976 lambda/r0 arcsec (e.g., Racine 1996, PASP 699, 108).
 
     @param lam_over_r0      The parameter that governs the scale size of the profile.
                             See above for details about calculating it.  [One of `lam_over_r0`,
-                            `fwhm`, or `half_light_radius` is required.]
+                            `fwhm`, `half_light_radius`, or both `lam` and `r0` is required.]
     @param fwhm             The full-width-half-max of the profile.  Typically given in arcsec.
-                            [One of `lam_over_r0`, `fwhm`, or `half_light_radius` is required.]
+                            [One of `lam_over_r0`, `fwhm`, `half_light_radius`, or both `lam` and
+                            `r0` is required.]
     @param half_light_radius  The half-light radius of the profile.  Typically given in arcsec.
-                            [One of `lam_over_r0`, `fwhm`, or `half_light_radius` is required.]
+                            [One of `lam_over_r0`, `fwhm`, `half_light_radius`, or both `lam` and
+                            `r0` is required.]
+    @param lam              Lambda (wavelength) in units of nanometers.  Must be supplied with
+                            `r0`, and in this case, image scales (`scale`) should be specified in
+                            units of `scale_unit`.
+    @param r0               The Fried parameter in units of meters.  Must be supplied with `lam`,
```

Comment:
When constructing the realistic atmospheric PSF code, I found it convenient to have a `r0_500` argument, the Fried parameter at 500nm.  This made it conceptually easier to investigate wavelength dependence of the PSF, since I could look through the same atmosphere at different wavelengths just by changing `lam` instead of needing to change both `lam` and `r0`.  I think having both options would be reasonable here (and in the real atmosphere code; need to do that).  I'll stop short of recommending a `lam_over_r0_500` parameter, though.  That seems too complicated.